lib/CLI/Parser.hs

```diff
@@ -318,9 +309,9 @@ tagOptions =
 -- Common
 ----------------------------------------------------------------------------
 
-readPath' :: Text -> Either String Path
+readPath' :: Text -> Either Text Path
```

Comment:
Oh, a bit unrelated, but can you please delete `_readEntryPath` while you're here? It's not used anywhere

Reply:
Done